src/actions/photos.js

```diff
@@ -0,0 +1,106 @@
+/**
+  Photos related features
+**/
+
+import cozy from 'cozy-client-js'
+
+import {
+  COZY_PHOTOS_DIR_ID,
+  FETCH_PHOTOS,
+  FETCH_PHOTOS_FAILURE,
+  RECEIVE_PHOTOS,
+  UPLOAD_PHOTOS,
+  UPLOAD_PHOTOS_SUCCESS,
+  UPLOAD_PHOTOS_SUCCESS_WITH_CONFLICTS,
+  UPLOAD_PHOTOS_FAILURE
+} from './constants'
```

Comment:
I understand, but it make more sense to get constants in a related file, even if some parts of code are more verbose. Those constants make no sense without actions. If tomorrow we remove the actions for whatever reason the constants may stay in constants.js. As @goldoraf agrees, let's avoid a constants file.